profile/rwahs.xml

```diff
@@ -7779,10 +7779,13 @@
         <permission group="memorial" access="read"/>
       </groupAccess>
       <bundlePlacements>
-        <placement code="preferred_labels">
-          <bundle>ca_object_labels.name</bundle>
+        <placement code="ca_objects_preferred_labels">
+          <bundle>ca_objects.preferred_labels</bundle>
           <settings>
             <setting name="label" locale="en_AU">Title</setting>
+            <setting name="format"><![CDATA[<nobr>^ca_objects.preferred_labels.name</nobr>]]></setting>
+            <setting name="delimiter"><![CDATA[]]></setting>
+            <setting name="maximum_length"><![CDATA[100]]></setting>
```

Comment:
These two settings (`delimiter` and `maximum_length`) came from modifying the display via the UI and exporting the profile.  Happy to remove them.